lib/DDG/Spice/Pubmed/Search.pm

```diff
@@ -0,0 +1,21 @@
+package DDG::Spice::Pubmed::Search;
```

Comment:
@b2ddg sorry, when the package name is 4+ levels deep, we take the 3rd+ portions and prefix the `alt_to` endpoints with that, creating `pubmed_abstract` and `pubmed_ids`.

If the package is just `DDG::Spice::Pubmed` we don't currently prefix the `alt_to`'s with anything (though maybe we should)?

As a fix, you can manually set the `alt_to` names to `pubmed_abstract` and `pubmed_ids`